_conferences/droidcon-pl-2015.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "Droidcon"
+website: http://droidcon.pl
+location: Kraków, PL
+
+date_start: 2015-12-04
+date_end:   2015-12-06
+
+cfp_site: http://goo.gl/forms/mfJxBfi7L3
```

Comment:
Unfortunately they haven't specified a CFP end date :-(
Should I assume a date, for instance start date of conference or delete cfp_site?